src/tools/pub/custom-package-repositories.md

```diff
@@ -220,6 +220,10 @@ your own custom package repository:
   <img src="/assets/img/tools/jfrog.svg" width="48" alt="JFrog logo">
   <a href="https://www.jfrog.com/confluence/display/JFROG/Pub+Repositories"><b>JFrog Artifactory</b></a>
 </li>
+<li>
+  <img src="onepub-logo.svg" width="48" alt="OnePub logo">
```

Comment:
Can you add the logo to [assets/img/tools/onepub-logo](https://github.com/dart-lang/site-www/tree/main/src/assets/img/tools) and then update this reference:

```suggestion
  <img src="/assets/img/tools/onepub-logo.svg" width="48" alt="OnePub logo">
```